src/core/lib/http/httpcli.cc

```diff
@@ -41,17 +41,59 @@
 #include "src/core/lib/iomgr/tcp_client.h"
 #include "src/core/lib/slice/slice_internal.h"
 
-struct internal_request {
+namespace {
```

Comment:
Please put this anonymous namespace inside of the `grpc_core` namespace.

Reply:
Done.